forecast-admin/forecast/opportunities/admin.py

```diff
@@ -1,20 +1,15 @@
 from django.contrib import admin
 
 # Register your models here.
-from .models import Award, Office
+from .models import Opportunity, Office
 
 
-@admin.register(Award)
-class AwardAdmin(admin.ModelAdmin):
-
-    # def get_form(self, request, obj=None, **kwargs):
-        # if not request.user.is_superuser:
-            # self.exclude = ('published',)
-        # return super(AwardAdmin, self).get_form(request, obj, **kwargs)
+@admin.register(Opportunity)
+class OpportunityAdmin(admin.ModelAdmin):
 
     def get_queryset(self, request):
         """Limit Pages to those that belong to the request's user."""
-        qs = super(AwardAdmin, self).get_queryset(request)
+        qs = super(OpportunityAdmin, self).get_queryset(request)
         if request.user.is_superuser:
```

Comment:
I think we want to change this to be is_superuser, or is in OSBU_EMPLOYEE group. Super users can delete things, and not all employees will likely have that access control.